examples/analytics-notification.amp.html

```diff
@@ -66,8 +66,8 @@
   <amp-user-notification
       layout=nodisplay
       id="amp-user-notification1"
-      data-show-if-href="http://localhost:8000/api/show?timestamp=TIMESTAMP"
```

Comment:
I don't know if "localhost" is really conveying the cross domain ability. One other compromise is to strip off the schema part and keep the domain name: "//localhost:8000". But that will still break if a different port is used.

Launching another local server might be too much for the purpose of demonstration. I personally prefer the relative path solution. On the other hand, we could always improve our documentation to cover those details.